tests/EmbeddedRelationsTest.php

```diff
@@ -945,4 +946,16 @@ public function testGetQueueableRelationsEmbedsOne()
         $this->assertEquals(['father'], $user->getQueueableRelations());
         $this->assertEquals([], $user->father->getQueueableRelations());
     }
+
+    public function testEmbedManySerialization()
+    {
+        $address = new Address(['country' => 'France']);
+        $address->save();
+
+        $address->addresses()->create(['city' => 'Paris']);
+        $address->addresses()->create(['city' => 'Nice']);
+
+        $results = $address->toArray();
+        $this->assertIsString($results['addresses'][0]['_id']);
```

Comment:
Unfortunately, this test does not cover the corrected bug. I've just checked, even if your fix is removed, the test still passes.